### ISSUE TITLE
ignored_variables should be converted to dict for better performance

### ISSUE THREAD

**Issue by ldperron** [Ouranosinc/pyPavics#21]
Can't tell for now if negligible or not. 
https://github.com/Ouranosinc/pyPavics/blob/23a8732b2181e3bafabc5ba8024b49f95277ed4f/pavics/catalog.py#L270

**Comment by dbyrns** [comment]
I agree with you. We should just make sure that the parameter ignored_variables stay a list (to preserve the interface) but convert it to a dict internally before using it in the loop.
Not a priority though.
